### PR TITLE
2

### PR DESCRIPTION
**Pull request by sulatskov34an**

### CODE REVIEW

File: app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt [hunk 1]
```diff
@@ -12,6 +12,7 @@ open class BasePresenter<V: BaseViewInterface> : BasePresenterInterface<V> {
         this.view = view
     }
```
Comment (sulatskov34an) [inline app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt]: zxcvzxcvzxvczxcvxzcv

File: app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt [hunk 2]
```diff
@@ -12,6 +12,7 @@ open class BasePresenter<V: BaseViewInterface> : BasePresenterInterface<V> {
...
 
+
     override fun detach() {
```
Comment (sulatskov34an) [inline app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt]: qwerqwerqwerqwer

File: app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt (hunk 1 above)
Comment (sulatskov134an) [reply app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt]: qwer

File: app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt (hunk 2 above)
Comment (sulatskov134an) [reply app/src/main/java/ru/glabrion/base/presenter/BasePresenter.kt]: qwerqwre

**Pull request closed without merging**